DuckDuckGo/FileDownload/View/DownloadsViewController.swift

```diff
@@ -479,6 +526,52 @@ extension DownloadsViewController: NSTableViewDataSource, NSTableViewDelegate {
 
 }
 
+enum DownloadsErrorViewType {
+    case openHelpURL
+    case openSystemSettings
+
+    var errorMessage: String {
+        return UserText.downloadsErrorMessage
+    }
+
+    var title: String {
+        switch self {
+        case .openHelpURL: return UserText.downloadsErrorSandboxCallToAction
+        case .openSystemSettings: return UserText.downloadsErrorNonSandboxCallToAction
+        }
+    }
+
+    @MainActor func onAction() {
+        switch self {
+        case .openHelpURL:
```

Comment:
I would suggest that we close downloads popover here (at least in the Sandbox variant where we open a new tab but the popover stays on top).

Reply:
Good call.